jgrapht-core/src/test/java/org/jgrapht/generate/RandomRegularGraphGeneratorTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * (C) Copyright 2018, by Emilio Cruciani and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+
+package org.jgrapht.generate;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.jgrapht.Graph;
+import org.jgrapht.alg.util.IntegerVertexFactory;
+import org.jgrapht.graph.DefaultDirectedGraph;
+import org.jgrapht.graph.DefaultEdge;
+import org.jgrapht.graph.Pseudograph;
+import org.jgrapht.graph.SimpleGraph;
+import org.junit.Test;
+
+/**
+ * @author Emilio Cruciani
+ * @since March 2018
+ */
+public class RandomRegularGraphGeneratorTest {
+
+    @Test
```

Comment:
For simple exception tests like this, we recommend using the "expected" parameter to the \@Test annotation.

https://github.com/junit-team/junit4/wiki/exception-testing

Reply:
Thanks, I did not know about it.